### PR TITLE
update table in README

### PR DESCRIPTION
**Pull request by chharvey**
I added "Resolve Value" and "Reject Value" columns to the table for clarity. Also switched rows around, for easier pattern recognition.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ This proposal is at stage 0 of [the TC39 process](https://tc39.github.io/process
...
 
-| name                 | description                                     |                                                                             |
-| -------------------- | ----------------------------------------------- | --------------------------------------------------------------------------- |
-| `Promise.allSettled` | does not short-circuit                          | [separate proposal](https://github.com/tc39/proposal-promise-allSettled) 🔜 |
-| `Promise.all`        | short-circuits when an input value is rejected  | added in ES2015 ✅                                                          |
-| `Promise.race`       | short-circuits when an input value is settled   | added in ES2015 ✅                                                          |
-| `Promise.any`        | short-circuits when an input value is fulfilled | this proposal 🆕                                                            |
+| name                 | resolve value                              | reject value                               | short-circuits?                  | spec                                                                        |
```
Comment (ljharb) [inline README.md]: typically i would expect this to be called “fulfillment value” and “rejection reason”
```suggestion
| name                 | fulfillment value                          | rejection reason                           | short-circuits?                  | spec                                                                        |
```

File: README.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ This proposal is at stage 0 of [the TC39 process](https://tc39.github.io/process
...
+| -------------------- | ------------------------------------------ | ------------------------------------------ | -------------------------------- | --------------------------------------------------------------------------- |
+| `Promise.allSettled` | array of all settled input values          | (does not reject)                          | (does not short-circuit)         | [separate proposal](https://github.com/tc39/proposal-promise-allSettled) 🔜 |
+| `Promise.race`       | first settled input value, if it resolved  | first settled input value, if it rejected  | when an input value is settled   | added in ES2015 ✅                                                           |
```
Comment (ljharb) [inline README.md]:
```suggestion
| `Promise.race`       | first settled input value, if it fulfilled  | first settled input value, if it rejected  | when an input value is settled   | added in ES2015 ✅                                                           |
```

File: README.md [hunk 3]
```diff
@@ -8,12 +8,12 @@ This proposal is at stage 0 of [the TC39 process](https://tc39.github.io/process
...
+| `Promise.all`        | array of all input values, if all resolved | first rejected input value                 | when an input value is rejected  | added in ES2015 ✅                                                           |
```
Comment (ljharb) [inline README.md]:
```suggestion
| `Promise.all`        | array of all input values, if all fulfilled | first rejected input reason                 | when an input value is rejected  | added in ES2015 ✅                                                           |
```

File: README.md [hunk 4]
```diff
@@ -8,12 +8,12 @@ This proposal is at stage 0 of [the TC39 process](https://tc39.github.io/process
...
+| `Promise.any`        | first resolved input value                 | array of all input values, if all rejected | when an input value is resolved  | this proposal 🆕                                                             |
```
Comment (ljharb) [inline README.md]:
```suggestion
| `Promise.any`        | first fulfilled input value                 | array of all rejection reasons, if all rejected | when an input value is resolved  | this proposal 🆕                                                             |
```

**Review by ljharb — COMMENTED** [review]
Feel free not to take the exact suggestions, but “resolved” isn’t the correct term - it’s “fulfilled”, and a rejected promise has a “reason” not a “value”.

**Review by gibson042 — COMMENTED** [review]
I think this change would make the motivation table more complete but less clear. The point is to show how this proposal fills a gap, but that gets lost in the details of much less relevant settlement result values.

**Comment by mathiasbynens** [comment]
Thanks for the suggestion, @chharvey!

As @gibson042 points out, the table is only intended to convey the gap filled by the `Promise.any` proposal, and not to be exhaustively informative. As such, I’m gonna close this PR.

**Pull request closed without merging**